BBallStatsTrack/src/com/example/bballstatstrack/models/Game.java

```diff
@@ -0,0 +1,406 @@
+package com.example.bballstatstrack.models;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import com.example.bballstatstrack.models.game.GameLog;
+import com.example.bballstatstrack.models.gameevents.GameEvent;
+import com.example.bballstatstrack.models.gameevents.TurnoverEvent;
+import com.example.bballstatstrack.models.gameevents.foulevents.NonShootingFoulEvent;
+import com.example.bballstatstrack.models.gameevents.foulevents.ShootingFoulEvent;
+
+import android.text.format.DateFormat;
+
+public class Game
```

Comment:
There can be multiple instances of a Game. After a game has been recorded, it is stored in the memory. This can be access later and the final state reviewed by the user,